Modules/posixmodule.c

```diff
@@ -274,8 +274,9 @@ corresponding Unix manual entries for more information on calls.");
 #  undef HAVE_SCHED_SETAFFINITY
 #endif
 
-#if defined(HAVE_SYS_XATTR_H) && defined(__GLIBC__) && !defined(__FreeBSD_kernel__) && !defined(__GNU__)
+#if defined(HAVE_SYS_XATTR_H) && defined(__linux__) && !defined(__FreeBSD_kernel__) && !defined(__GNU__)
 #  define USE_XATTRS
+#  include <linux/limits.h>
```

Comment:
is this specific direct linux kernel include required (it _should_ be harmless), i'd expect `sys/xattr.h` (below) or even `<limits.h>` to include this kernel header for us?  (as seems to be the case on glibc and android bionic libc?)

Reply:
It's needed for `XATTR_SIZE_MAX`. On a musl system, neither `<limits.h>` or `<sys/xattr.h>` pull that in. Is that okay, or do you want me to see about a configure check or something? Or try it as-is then adjust if required?

Reply:
lets just go with it as is, i added a comment to that effect for future reference.

Reply:
Cheers - happy to help with any issues if they do pop up, but I don't imagine any.